operators/multiclusterobservability/manifests/base/config/metrics_allowlist.yaml

```diff
@@ -300,3 +300,63 @@ data:
         expr: sum(rate(node_network_receive_drop_total{job=\"node-exporter\", device!=\"lo\"}[1m])) without(device)
       - record: instance:node_network_transmit_drop_excluding_lo:rate1m
         expr: sum(rate(node_network_transmit_drop_total{job=\"node-exporter\", device!=\"lo\"}[1m])) without(device)
+  metrics_rules.yaml: |
+    - name: NamespaceHighCPUUsage
+      expr: (sum(kube_resourcequota{resource=\"limits.cpu\", type=\"used\"}) by (namespace, resourcequota) / sum(kube_resourcequota{resource=\"limits.cpu\", type=\"hard\"}) by (namespace, resourcequota)) * 100 > 70
+      for: 2m
+      metrics_list:
+        - container_cpu_cfs_periods_total
+        - container_cpu_cfs_throttled_periods_total
+        - kube_pod_container_resource_limits 
+        - kube_pod_container_resource_requests 
+        - kube_resourcequota 
+        - mixin_pod_workload 
+        - namespace_workload_pod:kube_pod_owner:relabel 
+        - node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate 
+        - node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate 
+
+    - name: NamespaceHighMemoryUsage
+      expr: (sum(kube_resourcequota{resource=\"limits.memory\", type=\"used\"}) by (namespace, resourcequota) / sum(kube_resourcequota{resource=\"limits.memory\", type=\"hard\"}) by (namespace, resourcequota)) * 100 > 70
+      for: 2m
+      metrics_list:
+        - container_memory_cache 
+        - container_memory_rss 
+        - container_memory_swap 
+        - container_memory_working_set_bytes 
+        - kube_pod_container_resource_limits 
+        - kube_pod_container_resource_requests 
+        - kube_resourcequota 
+        - mixin_pod_workload 
+        - namespace_workload_pod:kube_pod_owner:relabel 
```

Comment:
For namespace collect rule, seems we need more enhancements.
e.g. in a large cluster, if we detect some problem in namespace A, should only start to collect the metrics in namespace A, not other namespaces.
If that, the metrics_list should be changed to something like below:
```
    matches:
      - __name__="container_memory_cache",namespace="{{ $labels.namespace }}"
```
by the way, to be consistent with allowlist, the metrics_list in SNO rules should change to "names"

Reply:
@marcolan018 I updated the metrics list, can you confirm if the changes are correct?